packages/migrations/README.md

```
@@ -1,3 +1,38 @@
 # @pulse/migrations
 
 Unified TypeScript migration runner for the Pulse PostgreSQL database.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)